index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Personal Portfolio Webpage</title>
+  </head>
+  <body>
+    <nav id="navbar">
+      <ul>
+        <li>
+          <a href="#welcome-section">Projects</a>
+        </li>
+      </ul>
+    </nav>
+    <section id="welcome-section">
+      <h1>Welcome to my portfolio</h1>
+    </section>
+
+    <a
+      rel="noopener"
+      id="profile-link"
+      href="https://github.com/omarbabou"
+      target="_blank"
+    ></a>
```

Comment:
Please use <a> tag inside an appropriate tag for the best practice of semantic HTML tags.

Reply:
**[OPTIONAL]**

- Great using the hint from the previous reviewer. You got that but theres a little issue with it. 
As of right now, your review  :point_right: **[Please use a tag inside an appropriate tag for the best practice of semantic HTML tags.]** :point_left: - is not explicit enough. It's still unclear to the code author as they don't really know what you mean. You should be able to tell the code author what appropriate tag to use to wrap the `a` tag. The word `tag` could refer to many tags in html. You being able to specify the appropriate tag and give instances of usage will make your review process better and will make you a good code reviewer. 

_**weldone**_ and ***Keep rocking!***